Snowplow/Internal/Configurations/SPEmitterConfiguration.h

```diff
@@ -0,0 +1,66 @@
+//
+//  SPEmitterConfiguration.h
+//  Snowplow
+//
+//  Created by Alex Benini on 03/12/2020.
+//  Copyright © 2020 Snowplow Analytics. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "SPConfiguration.h"
+#import "SPNetworkConnection.h"
+#import "SPEventStore.h"
+
+/*!
+ @brief An enum for buffer options.
+ */
+typedef NS_ENUM(NSUInteger, SPBufferOption) {
+    /**
+     * Sends both GET and POST requests with only a single event.  Can cause a spike in
+     * network traffic if used in correlation with a large amount of events.
+     */
+    SPBufferOptionSingle = 1,
+    /**
+     * Sends POST requests in groups of 10 events.  This is the default amount of events too
+     * package into a POST.  All GET requests will still emit one at a time.
+     */
+    SPBufferOptionDefaultGroup = 10,
+    /**
+     * Sends POST requests in groups of 25 events.  Useful for situations where many events
+     * need to be sent.  All GET requests will still emit one at a time.
+     */
+    SPBufferOptionHeavyGroup = 25
```

Comment:
`SPBufferOptionLargeGroup` might be a better name here

Reply:
It's the legacy name but I think I can deprecate it and add the new one.